web/home/src/pages/Home/home.tsx

```diff
@@ -136,15 +134,11 @@ function Home() {
         }
       });
   };
-
-  const defineGridStyle = (length: number) => {
-    let styles = gridStyle;
-    return (styles += length <= 3 ? ' grid-cols-3 ' : setResponsiveGrid);
+  const defineGridStyle = () => {
+    const setResponsiveGrid = `max-sm:grid-cols-1 sm:grid-cols-2 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4`;
+    const gridStyle = 'grid grid-flow-row mx-auto mt-3 gap-4';
+    return gridStyle + ' ' + setResponsiveGrid;
```

Comment:
why a function ? it returns a constant. Just use a constant

Reply:
you're right it was not like this at the beginning !